src/utils/player-error.js

```diff
@@ -1,38 +1,765 @@
+     */
+    'CAST_CANCELED_BY_USER': 8004,
+
+    /**
+     * The cast connection timed out.
+     * <br> error.data[0] is an error object from the Cast SDK.
+     */
+    'CAST_CONNECTION_TIMED_OUT': 8005,
+
+    /**
+     * The requested receiver app ID does not exist or is unavailable.
+     * Check the requested app ID for typos.
+     * <br> error.data[0] is an error object from the Cast SDK.
+     */
+    'CAST_RECEIVER_APP_UNAVAILABLE': 8006,
+
+
+    /**
+     * Offline storage is not supported on this browser; it is required for
+     * offline support.
+     */
+    'STORAGE_NOT_SUPPORTED': 9000,
+
+    /**
+     * An unknown error occurred in the IndexedDB.
+     * <br> On Firefox, one common source for UnknownError calls is reverting
+     * Firefox to an old version. This makes the indexedDB storage inaccessible
+     * for older versions. The only way to fix this is to delete the storage
+     * data in your profile. See https://goo.gl/eKVPPe.
+     * <br> error.data[0] is the error object.
+     */
+    'INDEXED_DB_ERROR': 9001,
+
+    /**
+     * The operation was aborted.  For example, by a call to destroy().
+     */
+    'OPERATION_ABORTED': 9002,
+
+    /**
+     * The specified item was not found in the IndexedDB.
+     * <br> error.data[0] is the offline URI.
+     */
+    'REQUESTED_ITEM_NOT_FOUND': 9003,
+
+    /**
+     * A network request was made with a malformed offline URI.
+     * <br> error.data[0] is the URI.
+     */
+    'MALFORMED_OFFLINE_URI': 9004,
+
+    /**
+     * The specified content is live or in-progress.
+     * Live and in-progress streams cannot be stored offline.
+     * <br> error.data[0] is the URI.
+     */
+    'CANNOT_STORE_LIVE_OFFLINE': 9005,
+
+    /**
+     * There is already a store operation in-progress, wait until it completes
+     * before starting another.
+     */
+    'STORE_ALREADY_IN_PROGRESS': 9006,
+
+    /**
+     * The specified manifest is encrypted but does not specify any init data.
+     * Without init data specified in the manifest, the content will not be
+     * playable offline.
+     * <br> error.data[0] is the URI.
+     */
+    'NO_INIT_DATA_FOR_OFFLINE': 9007,
+
+    /**
+     * shaka.offline.Storage was constructed with a Player proxy instead of a
+     * local player instance.  To fix this, use Player directly with Storage
+     * instead of the results of CastProxy.prototype.getPlayer().
+     */
+    'LOCAL_PLAYER_INSTANCE_REQUIRED': 9008,
+
+    /**
+     * When the manifest contains no period playable streams, it means the
+     * manifest is unsupported by the browser.
+     */
+    'CONTENT_UNSUPPORTED_BY_BROWSER': 9009
+
+  }
+
+
+  static _logger: any = getLogger('Error');
+
+  severity: any;
+  category: any;
+  code: any;
+  data: Object;
+
+  constructor(severity: number, category: number, code: number, data: Object = {}) {
+    this.severity = severity;
+    this.category = category;
+    this.code = code;
+    this.data = data;
```

Comment:
We have places in the code where we are sending this param as `{info: '...'}` or `{data: '...'}`.
The result to access this field we look like `error.data.data...` or `error.data.info...`
Need to manage properly the objects we are send to it.